docs/playbooks/wind-river-cloud-platform-deployment-manager.yaml

```diff
@@ -426,17 +453,94 @@
             - "Pod log: {{get_logs_pre_unlock.stdout}}"
             - "err: {{get_logs_pre_unlock.stderr}}"
 
-        # If the task "Wait until unlock task triggered" failed, we export some
-        # useful information to the fail playbook msg.
-        - name: Output dm logs
+        # if unlock was triggered, wait until unlock complete
+        # and check resources and logs.
+        - block:
+          - name: Waiting for port to become open
+            local_action:
+              module: wait_for
+                port={{ ansible_port }}
+                host={{ ansible_host }}
+                delay={{ boot_wait_time }}
+                timeout={{ wait_for_timeout }}
+                state=started
+            register: waiting_unlock
+            retries: 40
+            delay: 20
+            ignore_errors: yes
+
+          # Waiting task after unlock to catch right status
+          - wait_for:
+              timeout: 600
+              msg: Waiting after unlock due to apply DM config
```

Comment:
Do we wait for 600 sec after port open? Is there alternative way? Like retry the following "Retrieve kubectl resources reconciled status" for 600 seconds? I'd like to pass asap when reconcile goes fine.

Reply:
The arbitrary waiting avoid some errors due to not available resources.
In this update:
- Reduced this arbitrary time.
- Added loop to check resources as you suggested me.
That allows to have everything available at the check moment and finish it as soon as possible, but also waiting more time if needed.